lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -2041,7 +2042,13 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
   }
 
   void visitMacroExpansionDecl(MacroExpansionDecl *MED) {
-    llvm_unreachable("FIXME: macro expansion decl not handled in DeclChecker");
+    auto expanded = evaluateOrDefault(
+        Ctx.evaluator, ExpandMacroExpansionDeclRequest{MED}, {});
+    // FIXME: Handle this in name lookup instead.
```

Comment:
Definitely agreed that this should be handled in name lookup. This won't work across multiple source files.